inbm/dispatcher-agent/dispatcher/packagemanager/package_manager.py

```diff
@@ -395,9 +396,12 @@ def get(url: CanonicalUri,
             repo.add_from_requests_response(
                 urlparse(url.value).path.split('/')[-1], response, umask=umask)
     except HTTPError as e:
-        if e.response:
+        if e.response is not None:
```

Comment:
<h3>$\color{#FFD700}Coding Style: CodeSimplicity$</h3>

Instead of checking `if e.response is not None`, you can simplify the condition to `if e.response:` which is more Pythonic.